docs/OLD_TO_INCLUDE/wannier90/wannier90.rst

```diff
@@ -127,22 +127,22 @@ Outputs
 -------
 * output_parameters :py:class:`ParameterData <aiida.orm.data.parameter.ParameterData>` (accessed by ``calculation.res``). Contains the scalar properties. Currently parsed parameters include:
 
-  * ``number_wannier_functions``: the number of wannier functions
+  * ``number_wfs``: the number of wannier functions
   * ``Omega_D``, ``Omega_I``, ``Omega_OD``, ``Omega_total`` wich are: the diagonal :math:`\Omega_D`,
     invariant  :math:`\Omega_I`, offdiagonal :math:`\Omega_{OD}`, and total spread :math:`\Omega_{total}`. Units are always Ang^2
   * ``wannier_functions_output`` a list of dictionaries containing:
 
-    - coordinates: the center of the wannier function
-    - spread: the spread of the wannier function. Units are always Ang^2
-    - wannier_function: numerical index of the wannier function
+    - wf_centres: the center of the wannier function
+    - wf_spreads: the spread of the wannier function. Units are always Ang^2
+    - wf_ids: numerical index of the wannier function
     - im_re_ratio: if available the Imaginary/Real ratio of the wannier function
 
   * ``Warnings``: parsed list of warnings
```

Comment:
Here it should be lowercase `warnings` I think.

Reply:
True. Thanks!